Source/UI/CommonUI/UIStyle/CKOUIStyle+UIColor.swift

```diff
@@ -0,0 +1,66 @@
+import UIKit
+
+public extension CKOUIStyle {
+
+    /// Custom Color for Checkout UI style
+    enum Color {
```

Comment:
Please open a task to review on completion whether any of the old colours or fonts, sizes, etc are left behind.